prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -453,21 +453,23 @@ abstract class CoolTest {
         val hl7Sender = settings.findSender("$orgName.$hl7SenderName")
             ?: error("Unable to find sender $hl7SenderName for organization ${org.name}")
 
-        val allGoodReceivers = settings.receivers.filter {
-            it.organizationName == orgName &&
-                !it.name.contains("FAIL") &&
-                !it.name.contains("BLOBSTORE") &&
-                !it.name.contains("QUALITY") &&
-                !it.name.contains("AS2") &&
-                !it.name.contains("OTC")
-        }
-        val allGoodCounties = allGoodReceivers.map { it.name }.joinToString(",")
-
-        val csvReceiver = allGoodReceivers.filter { it.name == "CSV" }[0]
-        val hl7Receiver = allGoodReceivers.filter { it.name == "HL7" }[0]
-        val hl7BatchReceiver = allGoodReceivers.filter { it.name == "HL7_BATCH" }[0]
-        val redoxReceiver = allGoodReceivers.filter { it.name == "REDOX" }[0]
-        val hl7NullReceiver = allGoodReceivers.filter { it.name == "HL7_NULL" }[0]
+        val csvReceiver = settings.receivers.filter { it.organizationName == orgName && it.name == "CSV" }[0]
+        val hl7Receiver = settings.receivers.filter { it.organizationName == orgName && it.name == "HL7" }[0]
+        val hl7BatchReceiver = settings.receivers.filter { it.organizationName == orgName && it.name == "HL7_BATCH" }[0]
+        val redoxReceiver = settings.receivers.filter { it.organizationName == orgName && it.name == "REDOX" }[0]
+        val hl7NullReceiver = settings.receivers.filter { it.organizationName == orgName && it.name == "HL7_NULL" }[0]
+
+        lateinit var allGoodReceivers:  List<Receiver>
+        lateinit var allGoodCounties: String
+
+        fun initListOfGoodReceiversAndCounties(env: ReportStreamEnv) {
```

Comment:
Consider this instead:
```kotlin
fun initListOfGoodReceiversAndCounties(env: ReportStreamEnv) {
    allGoodReceivers = mutableListOf(csvReceiver, hl7Receiver, hl7BatchReceiver, hl7NullReceiver)
    if (env == ReportStreamEnv.LOCAL){
        allGoodReceivers.add(redoxReceiver)
    }

    allGoodCounties = allGoodReceivers.map { it.name }.joinToString(",")
}
```

Increased readability, clearer "these are the defaults and this is what changes in `LOCAL`"
Also, L466 will fail KTLint (indentation!)

Reply:
Changed.